posts/a-beginners-guide-to-influxdb.md

```diff
@@ -0,0 +1,109 @@
+---
+authors:
+- Ketki Deshpande
+tags:
+- Data
+date: 
+title: "A beginner’s guide to InfluxDB - A time series database"
+image: 
+---
+
+A time series database (TSDB) is specially made for data that can be evaluated as a time-series. Data like sensor outputs, market trends, cpu utilization or application performance metrics are some of the examples of data which can be best analyzed over a time period (a time-series). Most popular types of databases (SQL or No-SQL databases) are not optimized for handling large scale scans, summarization or aggregation of time-series data. Time-series databases have built-in functionality to work with aggregation, down-sampling, data lifecycle management, summarization and so on of time-series data. Let’s look at one of the popular time-series databases, InfluxDB. InfluxDB uses line protocol for sending and storing the data.
+
+# Line Protocol
+The line protocol is the text-based format that uses below syntax to write points to InfluxDB.
+
+``` <measurement>[,<tag_key>=<tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=<field_value>] [<timestamp>] ```
+
+The above syntax is used to insert a single point to an InfluxDB database. Since InfluxDB is a time-series database, the **point** here is analogous to a point on a graph that is representing a time-series data. And the collection of points is called a **series**. Now, let’s look at all the four components of the point.
+
+The first component, **Measurement** for an InfluxDB is similar to a Table for a SQL database. Measurement holds all the tags, fields and the time values and it makes sense to name the measurement such that it describes the data contained in it.
```

Comment:
The first component, **Measurement** for an InfluxDB, is similar to a Table for a SQL database.